app/views/layouts/_footer.html.haml

```diff
@@ -13,16 +13,16 @@
           - if Rails.env.development?
             .item= link_to "Dev Auth", auth_path(:developer)
 
-          .item= link_to "Firefox Extension", "https://github.com/transientBug/firefox-extension"
+          .item= link_to "Firefox Extension", "https://github.com/transientBug/firefox-extension", data: { turbolinks: false }
           .item= link_to "Images", images_path
 
       .three.wide.column
         %h4.ui.inverted.header All the Credits!
         .ui.inverted.link.list
-          .item=link_to "Semantic UI", "https://semantic-ui.com/"
-          .item= link_to "Rails 5.2", "https://github.com/rails/rails"
+          .item=link_to "Semantic UI", "https://semantic-ui.com/", data: { turbolinks: false }
+          .item= link_to "Rails 5.2", "https://github.com/rails/rails", data: { turbolinks: false }
           .item
-            = link_to "https://github.com/transientBug/transientbug-rails" do
+            = link_to "https://github.com/transientBug/transientbug-rails", data: { turbolinks: false } do
```

Comment:
ow. I'm surprised turbolinks isnt smarter about this

Reply:
dude, the sadness